traffic_ops/traffic_ops_golang/server/servers.go

```diff
@@ -1477,7 +1477,7 @@ func Update(inf *api.APIInfo) (int, error, error) {
 
 	where := `WHERE s.id = $1`
 	var selquery string
-	if inf.Version.Major <= 4 {
+	if !inf.Version.GreaterThan(api.Version{Major: 4}) {
```

Comment:
Back to the original line in question: `!Version.GreaterThan` is not as easy to reason about as `Version.LessThanOrEqualTo`. But if you feel strongly about it, `!Version.GreaterThan` it's fine, since no `Version.LessThanOrEqualTo` method already exists.